contrib/html/html.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import defineFunction, {ordargument} from "../../src/defineFunction";
+import buildCommon from "../../src/buildCommon";
+import mathMLTree from "../../src/mathMLTree";
+import domTree from "../../src/domTree";
+import {defineGroupParser, newArgument} from "../../src/Parser";
+import ParseNode from "../../src/ParseNode";
+
+import {buildExpression as htmlBuildExpression} from "../../src/buildHTML";
+import {buildExpression as mmlBuildExpression} from "../../src/buildMathML";
+
+import type {ParsedArg} from "../../src/Parser";
+
+defineGroupParser("string", function(optional: boolean): ?ParsedArg {
+    const res = this.parseStringGroupWithBalancedBraces("string", optional);
+    if (!res) {
+        return null;
+    }
+    return newArgument(new ParseNode("string", res.text, "text"), res);
+});
+
+// $FlowFixMe
+defineFunction({
+    type: "class",
+    names: ["\\class"],
+    props: {
+        numArgs: 2,
+        allowedInText: true,
+        greediness: 3,
+        argTypes: ["string", "original"],
+    },
+    handler: (context, args) =>  {
+        const classes = args[0];
+        const body = args[1];
+        return {
+            type: "class",
+            classes: classes.value,
+            value: ordargument(body),
+        };
+    },
+    htmlBuilder: (group, options) => {
+        const elements = htmlBuildExpression(group.value.value, options, false);
+        const classes = group.value.classes.trim().split(/\s+/);
+        const fragment = new buildCommon.makeFragment(elements);
+
+        fragment.children.forEach(children => {
+            if (!(children instanceof domTree.svgNode)) {
+                children.classes.push(...classes);
+            }
+        });
```

Comment:
I would've expected all of the elements to be wrapped in a single `span` with the given class applied.  This applies the class to each element.  Is that what MathJax does?

Reply:
@kevinbarabash No, MathJax does, what you said:
>all of the elements to be wrapped in a single `span` with the given class applied.

Though, I'm not sure which is more suitable. For warraping in a `span`, I think we can do in a similar way to `\href`:
https://github.com/Khan/KaTeX/blob/2a2742453b86fba6fd5184efb3c3255400796fb7/src/functions/href.js#L25-L66

Reply:
Blocked on #1070

Reply:
#1070 has been merged, but I think we should probably do what MathJax does since that's probably the only other TeX renderer which supports `\class`.